src/main/kotlin/org/springframework/security/dsl/config/builders/servlet/headers/CacheControlDsl.kt

```diff
@@ -26,8 +26,11 @@ import org.springframework.security.config.annotation.web.configurers.HeadersCon
  * @author Eleftheria Stein
```

Comment:
Optionally, if you would like to, you can add your name to the list of authors for all the classes that you have updated.